test/test_controllers/test_multicontroller.py

```diff
@@ -0,0 +1,147 @@
+from hdmf.testing import TestCase
+from ipywidgets import Button, Checkbox, Dropdown, ToggleButton
+
+from nwbwidgets.controllers import BaseController, MultiController
+
+
+class ExampleMultiController(MultiController):
+    def __init__(self, attributes: dict):
+        self.custom_attribute = "abc"
+        super().__init__(attributes=attributes)
+
+
+class ExampleControllerWithObservers(BaseController):
+    def setup_attributes(self):
+        self.button = Button()
+        self.check_box = Checkbox()
+
+    def setup_children(self):
+        self.children = (self.button, self.check_box)
+
+    def check_box_on_button_click(self, change):
+        if self.check_box.value is True:
+            self.check_box.value = False
+        elif self.check_box.value is False:
+            self.check_box.value = True
+
+    def setup_observers(self):
+        self.button.on_click(self.check_box_on_button_click)
```

Comment:
This block shows the intended design of most basic controllers using the 3-step pattern.

For the most part, overriding `__init__` is not necessary at all, just

(a) define your attributes

(b) define how you want the children to be arranged (could also consist of nested boxes if needed)

(c) define observers on the attributes (which could be methods of the class or locally defined ones in `setup_observers`)